ntpcheck/cmd/utils_linux.go

```diff
@@ -23,6 +23,25 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
```

Comment:
is this linux specific?

Reply:
It was used only in `timexToDesc` which in turn was only used in `utils_linux.go`. So linter was unhappy.